### DISCUSSION TITLE
Add links to private official documents using Tequila authentication

### DISCUSSION THREAD

**Discussion by Scrashdown** [clicepfl/clic-website#237] [Ideas]
Referring to #25, some documents with restricted access should also be made available on the site.

Access control should be implemented with Tequila authentication.

**Comment by mochiruntime** [comment]
There seems to be a passportjs strategy as an npm package for Tequila authentication : https://gitlab.com/epfl-sti/passport-tequila

Here's PassportJS doc : http://www.passportjs.org/
